script/configure-git-hooks.ps1

```diff
@@ -0,0 +1,22 @@
+# Install Chocolatey if not already installed
+if (-not (Get-Command choco -ErrorAction SilentlyContinue)) {
+    Set-ExecutionPolicy Bypass -Scope Process -Force
+    iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
```

Comment:
Gives an error:


C:\Users\foo\git\DTaaS [pr-580 ≡]> powershell.exe -File .\script\configure-git-hooks.ps1
Forcing web requests to allow TLS v1.2 (Required for requests to Chocolatey.org)
Getting latest version of the Chocolatey package for download.
Not using proxy.
Getting Chocolatey from https://community.chocolatey.org/api/v2/package/chocolatey/2.2.2.
Downloading https://community.chocolatey.org/api/v2/package/chocolatey/2.2.2 to C:\Users\foo\AppData\Local\Temp\chocolatey\chocoInstall\chocolatey.zip
Not using proxy.
Extracting C:\Users\foo\AppData\Local\Temp\chocolatey\chocoInstall\chocolatey.zip to C:\Users\foo\AppData\Local\Temp\chocolatey\chocoInstall
Installing Chocolatey on the local machine
WARNING: Setting ChocolateyInstall Environment Variable on USER and not SYSTEM variables.
  This is due to either non-administrator install OR the process you are running is not being run as an Administrator.
Creating ChocolateyInstall as an environment variable (targeting 'User')
  Setting ChocolateyInstall to 'C:\ProgramData\chocolatey'
WARNING: It's very likely you will need to close and reopen your shell
  before you can use choco.
Installation of Chocolatey to default folder requires Administrative permissions. Please run from elevated prompt. Please see https://chocolatey.org/instal
l for details and alternatives if needing to install as a non-administrator.
At C:\Users\foo\AppData\Local\Temp\chocolatey\chocoInstall\tools\chocolateysetup.psm1:291 char:9
+         throw "Installation of Chocolatey to default folder requires  ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Installation of...-administrator.:String) [], RuntimeException
    + FullyQualifiedErrorId : Installation of Chocolatey to default folder requires Administrative permissions. Please run from elevated prompt. Please se
   e https://chocolatey.org/install for details and alternatives if needing to install as a non-administrator.

Reply:
> Gives an error:
> 
> C:\Users\foo\git\DTaaS [pr-580 ≡]> powershell.exe -File .\script\configure-git-hooks.ps1 Forcing web requests to allow TLS v1.2 (Required for requests to Chocolatey.org) Getting latest version of the Chocolatey package for download. Not using proxy. Getting Chocolatey from https://community.chocolatey.org/api/v2/package/chocolatey/2.2.2. Downloading https://community.chocolatey.org/api/v2/package/chocolatey/2.2.2 to C:\Users\foo\AppData\Local\Temp\chocolatey\chocoInstall\chocolatey.zip Not using proxy. Extracting C:\Users\foo\AppData\Local\Temp\chocolatey\chocoInstall\chocolatey.zip to C:\Users\foo\AppData\Local\Temp\chocolatey\chocoInstall Installing Chocolatey on the local machine WARNING: Setting ChocolateyInstall Environment Variable on USER and not SYSTEM variables. This is due to either non-administrator install OR the process you are running is not being run as an Administrator. Creating ChocolateyInstall as an environment variable (targeting 'User') Setting ChocolateyInstall to 'C:\ProgramData\chocolatey' WARNING: It's very likely you will need to close and reopen your shell before you can use choco. Installation of Chocolatey to default folder requires Administrative permissions. Please run from elevated prompt. Please see https://chocolatey.org/instal l for details and alternatives if needing to install as a non-administrator. At C:\Users\foo\AppData\Local\Temp\chocolatey\chocoInstall\tools\chocolateysetup.psm1:291 char:9
> 
> * ```
>       throw "Installation of Chocolatey to default folder requires  ...
>   ```
> * ```
>       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>   ```
>   
>   
>       
>         
>       
>   
>         
>       
>   
>       
>     
>   
>   * CategoryInfo          : OperationStopped: (Installation of...-administrator.:String) [], RuntimeException
>   * FullyQualifiedErrorId : Installation of Chocolatey to default folder requires Administrative permissions. Please run from elevated prompt. Please se
>     e https://chocolatey.org/install for details and alternatives if needing to install as a non-administrator.

It seems that the script is attempting to install Chocolatey to the default location, which requires administrative permissions. However, the script is being run without elevated privileges.